wrangler-docs/directives/set-column.md

```diff
@@ -59,3 +59,42 @@ would result in this record:
   "salutation": "ROOT JOLTIE"
 }
 ```
+
+## Arithmetic and decimal operations example 
+
+Using this record as an example:
+```
+{
+  "first": "Root",
+  "last": "Joltie",
+  "age": "32",
+  "hrlywage": "11.79",
+  "yrlywage": "21151.26"
+  "yrlyhours": "1794"
+  "yrlybonus": "25000.98"
+  "wageincrease": "2.50"
+}
+```
+Applying these directives:
+```
+set-column newhrlywage arithmetic:add(hrlywage,wageincrease)
```

Comment:
should list all the functions available in the documentation

Reply:
All the new functions are included in the documentation in the last update.